pkg/aspect/config/quotas.proto

```diff
@@ -20,7 +20,16 @@ option go_package="config";
 
 // Configures a quotas aspect.
 message QuotasParams {
-    // A list of names of istio.mixer.v1.config.descriptor.QuotaDescriptors determining the
-    // set of quotas that will be checked for any Quota call.
-    repeated string quota_descriptor_names = 1;
+    message Quota {
+        // Must match the name of some quota_descriptor in the global config.
+        string descriptor = 1;
```

Comment:
I think we want to change the `metric` field to `descriptor_name` rather than `descriptor`, we should do it here too if so. The reasoning is that the protoc generated golang struct renamed `descriptor` to `descriptor_` because protoc thinks of `descriptor` as a reserved word.